go.mod

```diff
@@ -0,0 +1,118 @@
+	github.com/openshift/library-go v0.0.0-20191003152030-97c62d8a2901
+	github.com/openshift/machine-config-operator v0.0.0-20191029224459-e8050812d506
+	github.com/pborman/uuid v1.2.0
+	github.com/pkg/errors v0.8.1
+	github.com/pkg/sftp v1.10.0
+	github.com/shurcooL/httpfs v0.0.0-20190707220628-8d4bc4ba7749
+	github.com/shurcooL/vfsgen v0.0.0-20181202132449-6a9ea43bcacd
+	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/cobra v0.0.5
+	github.com/stoewer/go-strcase v1.1.0 // indirect
+	github.com/stretchr/testify v1.4.0
+	github.com/terraform-providers/terraform-provider-aws v0.0.0-00010101000000-000000000000
+	github.com/terraform-providers/terraform-provider-azurerm v1.37.0
+	github.com/terraform-providers/terraform-provider-google v0.0.0-20191204001720-5937ed8454f5
+	github.com/terraform-providers/terraform-provider-ignition v1.1.0
+	github.com/terraform-providers/terraform-provider-local v1.4.0
+	github.com/terraform-providers/terraform-provider-openstack v1.24.0
+	github.com/terraform-providers/terraform-provider-random v2.0.0+incompatible
+	github.com/vincent-petithory/dataurl v0.0.0-20160330182126-9a301d65acbb
+	golang.org/x/crypto v0.0.0-20191202143827-86a70503ff7e
+	golang.org/x/lint v0.0.0-20191125180803-fdd1cda4f05f
+	golang.org/x/net v0.0.0-20191204025024-5ee1b9f4859a // indirect
+	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
+	golang.org/x/sys v0.0.0-20191128015809-6d18c012aee9
+	golang.org/x/tools v0.0.0-20191204011308-9611592c72f6 // indirect
+	golang.org/x/xerrors v0.0.0-20191011141410-1b5146add898 // indirect
+	google.golang.org/api v0.13.0
+	gopkg.in/AlecAivazis/survey.v1 v1.8.7
+	gopkg.in/ini.v1 v1.42.0
+	gopkg.in/yaml.v2 v2.2.7
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
+	k8s.io/klog v1.0.0
+	k8s.io/utils v0.0.0-20191114184206-e782cd3c129f
+	sigs.k8s.io/cluster-api-provider-aws v0.0.0-00010101000000-000000000000
+	sigs.k8s.io/cluster-api-provider-azure v0.0.0-00010101000000-000000000000
+	sigs.k8s.io/cluster-api-provider-openstack v0.0.0
+)
+
+replace (
+	github.com/Azure/azure-sdk-for-go => github.com/hashicorp/azure-sdk-for-go v36.2.2-hashi+incompatible
+	github.com/coreos/ignition => github.com/coreos/ignition v0.28.0
+	github.com/go-log/log => github.com/go-log/log v0.1.1-0.20181211034820-a514cf01a3eb
+	github.com/metal3-io/baremetal-operator => github.com/openshift/baremetal-operator v0.0.0-20191017022404-1f983453b8d3
+	github.com/metal3-io/cluster-api-provider-baremetal => github.com/openshift/cluster-api-provider-baremetal v0.0.0-20190821174549-a2a477909c1d
+	github.com/terraform-providers/terraform-provider-aws => github.com/terraform-providers/terraform-provider-aws v0.0.0-20190510001811-4b894dbf13f6
+	github.com/terraform-providers/terraform-provider-ignition => github.com/LorbusChris/terraform-provider-ignition v1.0.2-0.20191203224725-42fde74592ae
```

Comment:
would like to see the reason we are using local fork?

Reply:
The link validation method was renamed in 0.28.0 until it was completely removed after that.
Bumping Ign to 0.28.0 requires this, see https://github.com/LorbusChris/terraform-provider-ignition/commit/42fde74592aec8e7df7d589247d5c62da2b6e805